tests/utilities/prompt.test.ts

```diff
@@ -0,0 +1,225 @@
+import * as dotenv from "dotenv";
+import Anthropic from "@anthropic-ai/sdk";
+import { MessageParam } from "@anthropic-ai/sdk/resources/messages";
+import { Tracer } from "@opentelemetry/sdk-trace-node";
+import { CohereClient } from "cohere-ai";
+import { Message as CohereMessage } from "cohere-ai/api";
+import OpenAI from "openai";
+
+import { Humanloop } from "../../src";
+import { ModelProviders } from "../../src/api/types/ModelProviders";
+import { PromptKernelRequest } from "../../src/api/types/PromptKernelRequest";
+import {
+    HUMANLOOP_FILE_KEY,
+    isHumanloopSpan,
+    readFromOpenTelemetrySpan,
+} from "../../src/otel";
+import { UtilityPromptKernel, promptUtilityFactory } from "../../src/utilities/prompt";
+import {
+    callLLMMessages,
+    openTelemetryHLProcessorTestConfiguration,
+    openTelemetryTestConfiguration,
+} from "./fixtures";
+
+// NOTE: Add here as more Providers are added
+const PROVIDER_AND_MODEL: [ModelProviders, string][] = [
+    [ModelProviders.Openai, "gpt-4o"],
+    [ModelProviders.Anthropic, "claude-3-opus-latest"],
```

Comment:
should use 3.5-haiku.

(tests should generally use the smaller models)

Reply:
ack fixed